lib/Data/Xls.hs

```diff
@@ -111,6 +162,27 @@ decodeXls file =
             count <- liftIO $ c_xls_wb_sheetcount pWB
             mapM_ (decodeOneWorkSheet file pWB) [0 .. count - 1]
 
+-- | A work-around via temporary files and 'decodeXlsIO', 
+-- since this lib is lacking a pure function to decode the contents 
+-- of an XLS file. 
```

Comment:
Can make this easier to understand for users.

"Since this library lacks a pure function to decode from a buffer, we just write the buffer to a temporary file and decode the file".